src/test/java/com/wikia/webdriver/testcases/discussions/PollsTests.java

```diff
@@ -0,0 +1,160 @@
+import com.wikia.webdriver.common.remote.discussions.DiscussionsClient;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.*;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(groups = "discussions-polls")
+public class PollsTests extends NewTestTemplate {
+
+    public static final int DEFAULT_ANSWERS_NUMBER = 2;
+
+    private void createInitialPostWithPoll() {
+        final PostEntity.Data data = DiscussionsClient.using(User.USER, driver).createPostWithUniqueData();
+
+    }
+
+    @Test
+    @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+    @Execute(asUser = User.USER_6, onWikia = MercuryWikis.DISCUSSIONS_2)
+    public void userCanCreatePostWithSimplePollOnDesktop() {
+        PostsListPage page = new PostsListPage().open();
+        BasePostsCreator postsCreator = page.getPostsCreatorDesktop();
+        postsCreator.click().closeGuidelinesMessage().clickAddCategoryButton().selectFirstCategory();
+        postsCreator.addTitleWith(TextGenerator.createUniqueText());
+        Poll poll = postsCreator.addPoll();
+        Assert.assertFalse(postsCreator.isPostButtonActive());
+
+        poll.addTitle(TextGenerator.createUniqueText());
+        Assert.assertFalse(postsCreator.isPostButtonActive());
+        poll.addNthAnswer(TextGenerator.createUniqueText(), 0);
+        Assert.assertFalse(postsCreator.isPostButtonActive()); //poll needs to have at least 2 answers options
+        poll.addNthAnswer(TextGenerator.createUniqueText(), 1);
+        postsCreator.clickSubmitButton();
+        page.waitForLoadingSpinner();
+
+        Assert.assertTrue(new PostsListPage().getPost().firstPostHasPoll());
+    }
+
+    @Test(dependsOnMethods = {"userCanCreatePostWithSimplePollOnDesktop"})
+    @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+    @Execute(asUser = User.ANONYMOUS, onWikia = MercuryWikis.DISCUSSIONS_2)
+    public void anonUserCanNotVoteInPollOnDesktop() {
+        Poll poll = new PostsListPage().open().getPost().clickNthPostWithPoll(0).getPoll();
+        poll.clickPollTitle();
+        SignInToFollowModalDialog signInModal = new SignInToFollowModalDialog();
+        Assert.assertTrue(signInModal.isVisible());
+
+        signInModal.clickOkButton();
+        poll.clickNthAnswer(0);
+        Assert.assertTrue(signInModal.isVisible());
+    }
+
+    @Test(dependsOnMethods = {"userCanCreatePostWithSimplePollOnDesktop"})
+    @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+    @Execute(asUser = User.USER_3, onWikia = MercuryWikis.DISCUSSIONS_2)
+    public void loggedInUserCanVoteOnceInPollOnDesktop() {
+        Poll poll = new PostsListPage().open().getPost().clickNthPostWithPoll(0).getPoll();
+        poll.clickNthAnswer(0);
+        poll.clickNthAnswer(1); //change answer before submitting a vote
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)